test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -22,15 +22,40 @@ import (
 	"testing"
 
 	"github.com/openzipkin/zipkin-go/model"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
 	"knative.dev/eventing/test/base/resources"
 	"knative.dev/eventing/test/common"
 	tracinghelper "knative.dev/eventing/test/conformance/helpers/tracing"
 )
 
-func BrokerTracingTestHelper(t *testing.T, channelTestRunner common.ChannelTestRunner) {
-	tracingTestHelper(t, channelTestRunner, setupBrokerTracing)
+// BrokerTracingTestHelperWithChannelTestRunner runs the Broker tracing tests for all Channels in
+// the ChannelTestRunner.
+func BrokerTracingTestHelperWithChannelTestRunner(t *testing.T, channelTestRunner common.ChannelTestRunner, setupClient SetupClientFunc) {
```

Comment:
Can we add line break here?

Reply:
Concretely, what do you want this to look like? A new line after the function definition? New lines inside the function definition to make this one very long line into multiple shorter ones?

Reply:
Sorry for being unclear... What I meant was breaking this long line into multiple shorter lines.

Reply:
Done.